api/raml/device-sdk.raml

```diff
@@ -0,0 +1,272 @@
+#%RAML 0.8
+---
+title: device-sdk
+version: "1.1.x"
+baseUri: "http://device-x:49999/api"
+protocols: [ HTTP ]
+
+documentation:
+    - title: Device Service API
+      content: This is the REST API implemented by device-sdk-go and device-sdk-c. It is therefore available on device services built with either SDK.
+
+schemas: 
+    -
+        responseobjects: '{"type":"array","$schema":"http://json-schema.org/draft-06/schema#","title":"responseobjects","items":{"type":"object","required":false,"$ref":"#/schemas/responseobject"}}'
+    -
+        responseobject: '{"type":"object","$schema":"http://json-schema.org/draft-06/schema#","title":"responseobject","properties": {"device":{"description":"the name of the device","type":"string"}, "origin":{"description":"timestamp indicating when the readings were taken","type":"integer"}, "readings":{"type":"array","items":{"type":"object","$ref":"#/schemas/reading"}}}}'
+    -
+        reading: '{"type":"object","$schema":"http://json-schema.org/draft-06/schema#","title":"reading","properties":{"name":{"type":"string"},"value":{"type":"string"}}}'
+    -
+        setting: '{"type":"object","$schema":"http://json-schema.org/draft-06/schema#","title":"setting","additionalProperties":{"type":"string"}}'
+    -
+        callbackalert: '{"type": "object","$schema": "http://json-schema.org/draft-06/schema#","title": "Notification Schema","properties": {"id": {"description": "the identifier of the object which is called back","type": "string"},"actionType": {"description": "the type of the called back object","enum":["PROFILE","DEVICE","PROVISIONWATCHER","SCHEDULE","SCHEDULEEVENT"],"type": "string"}},"required": ["id"]}'
+    -
+        versionobject: '{"type":"object","$schema":"http://json-schema.org/draft-06/schema#","title":"version","properties":{"version":{"type":"string"},"sdk_version":{"type":"string"}}}
```

Comment:
Missing a single quote on the end of the line.

Reply:
updated